src/pages/cluster-management/clusterInfo.html

```diff
@@ -72,6 +72,14 @@ <h1>
 
 <div ng-if="!loader" class="nodes">
     <div cluster-graphical-view></div>
+    <div id="toggle-legend-button" class="toggle-legend-btn">
+        <button class="btn btn-link"
+                ng-click="toggleLegend()"
+                tooltip="{{'cluster_management.cluster_page.toggle_legend_btn' | translate}}"
+                tooltip-placement="right">
+            <span class="icon-help legend-toggle-icon"></span>
```

Comment:
legend-toggle-icon -> toggle-legend-icon in order to be consistent with the button class

Reply:
changed